demo/src/index.html

```diff
@@ -17,6 +17,7 @@ <h2>AVR8js LED Demo</h2>
       <div class="toolbar">
         <button id="run-button">Run</button>
         <button id="stop-button" disabled>Stop</button>
+        <button id="revert-button">Back to leds example</button>
```

Comment:
LED is written [uppercase](https://en.wikipedia.org/wiki/Light-emitting_diode), so please change "leds" to "LEDs"

Reply:
ok